api/src/main/java/org/arquillian/cube/q/api/NetworkChaos.java

```diff
@@ -0,0 +1,138 @@
+package org.arquillian.cube.q.api;
```

Comment:
Generall thought - maybe it would be good to move high-level API bits to their own files rather than having them nested here?